commit/trk2dictionary/trk2dictionary.pyx

```diff
@@ -155,17 +155,21 @@ cpdef run( filename_trk, path_out, filename_peaks = None, filename_mask = None,
     if ret == 0 :
         print '   [ DICTIONARY not generated ]'
         return None
-    print '   [ %.1f seconds ]' % ( time.time() - tic )
 
     # create new TRK with only fibers in the WM mask
-    print '\t* generate tractogram: %s' % gen_trk
     if gen_trk :
+        print '\t* Generate tractogram: '
```

Comment:
I'd write something more specific, something like "Generate tractogram matching the dictionary", no?